agent/batch.go

```diff
@@ -0,0 +1,47 @@
+package agent
+
+import (
+	"errors"
+
+	"github.com/odpf/meteor/models"
+)
+
+type batch struct {
+	data []models.Record
+	size int
```

Comment:
This doesn't really reflect what it does. We should call it `capacity` I think.

Reply:
oh isnt batch size is a common word in this use case?

Reply:
But this size variable doesn't really reflect our current batch size, size is reflected using `len(data)`, this variable is currently used to check if its full or not, and its more of a capacity of this batch not size.

Reply:
oh truee! yeah that makes a lot of sense. updating it.